__tests__/integrations.test.js

```diff
@@ -53,11 +53,67 @@ describe('Topics', () => {
       .get('/api/topics')
       .expect(200)
       .then(({ body }) => {
+        const { topics } = body;
         const expectedKeys = ['slug', 'description'];
-        body.forEach((topic) => {
+        topics.forEach((topic) => {
           expect(Object.keys(topic)).toEqual(expectedKeys);
+          expect(Object.keys(topic).length).toBe(2);
+          expect(typeof topic.slug).toBe('string');
+          expect(typeof topic.description).toBe('string');
         });
       });
   });
   test.todo('Test for no topics found..?');
 });
+
+// ARTICLES
+describe('Articles', () => {
+  test('GET 200: Endpoint returns an a singular article by ID, with the appropriate properties.', () => {
+    return request(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        const expectedArticle = {
+          article_id: 1,
+          title: 'Living in the shadow of a great man',
+          topic: 'mitch',
+          author: 'butter_bridge',
+          body: 'I find this existence challenging',
+          created_at: '2020-07-09T20:11:00.000Z',
+          votes: 100,
+          article_img_url:
+            'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+        };
+        const expectedKeys = [
+          'article_id',
+          'title',
+          'topic',
+          'author',
+          'body',
+          'created_at',
+          'votes',
+          'article_img_url',
+        ];
+        expectedKeys.forEach((key) => {
+          expect(key in article).toBe(true);
+        });
+        expect(Object.keys(article).length).toBe(8);
+        expect(article).toMatchObject(expectedArticle);
+        expect(typeof article.article_id).toBe('number');
```

Comment:
a better jest matcher for testing objects, especially as you'll start testing bigger ones in the next few tickets, is .toMatchObject or .objectContaining. I'd suggest having a go at using them on future tickets